lib/sessions.ts

```diff
@@ -85,6 +109,98 @@ export class Sessions {
     });
   }
 
+  /** Parse a JWT and verify the signature, preferring local verification over remote.
+   *
+   * If maxTokenAge is set, remote verification will be forced if the JWT was issued at (based on
+   * the "iat" claim) more than maxTokenAge seconds ago.
+   */
+  async authenticate_jwt(
```

Comment:
I believe the rest of the library does camelCase, so I believe this should be authenticateJwt (see loginOrCreate)